complete/completeprocessing.php

```diff
@@ -1,146 +1,172 @@
 <?php
-	$title = 'Boiler Books';
-	$completeactive = "active";
-	include '../menu.php';
+    $title = 'Boiler Books';
+    $completeactive = "active";
+    include '../menu.php';
 ?>
 
-<?php
-include '../dbinfo.php';
-echo '<div class="container">';
-// define variables and set to empty values
-$cost = $comments = $receipt = $purchasedate = "";
-$committee = $_SESSION['committeec'];
-$item = $_SESSION['itemc'];
-$purchaseID = $_SESSION['purchaseIDc'];
-if ($purchaseID == '') {
-	echo 'no purchase';
-	header("Location: index.php");
-}
-else {
-	$sqler = $uploaderr = '';
-	$comments = test_input($_POST["comments"]);
-	$cost = test_input($_POST["cost"]);
-	$purchasedate = test_input($_POST["purchasedate"]);
-	$purchasedateorig = $purchasedate;
-
-	$purchasedate = str_replace('-','/',$purchasedate);
-	$purchasedate = date('Y-m-d H:i:s', strtotime($purchasedate));  
-	$purchasedatetemp = $purchasedate;
-	$receipt = $servername . "/";
-	$target_dir = "../receipts/";
-	$target_dir_save = "/receipts/";
-	$uploadOk = 1;
-	$FileType = pathinfo($target_dir . basename($_FILES["fileToUpload"]["name"]),PATHINFO_EXTENSION);
-	$target_file = $target_dir . $committee . "_" . $item . "_" . $purchaseID . "." . $FileType;
-	$target_file_save = $target_dir_save . $committee . "_" . $item . "_" . $purchaseID . "." . $FileType;
-	// Check if file already exists
-	if (file_exists($target_file)) {
-		$uploaderr = $uploaderr . " Your file already exists on the server<br>";
-		$uploadOk = 0;
-	}
-	// Check file size
-	// file size must be less than 2MB
-	if ($_FILES["fileToUpload"]["size"] > 2048000 || $_FILES["fileToUpload"]["size"] == 0) {
-		$uploaderr = $uploaderr . "Your reimbursement cert is larger than 2MB<br>";
-		$uploadOk = 0;
-	}
-	// Allow certain file formats
-	if($FileType != "pdf" and $FileType != "PDF" and $FileType != "jpg" and $FileType != "jpeg"and $FileType != "JPG"and $FileType != "JPEG") {
```

Comment:
Converted to putting the correct file types in an array, and using strtolower making it much easier to maintain and doesn't worry about weird capitalization.